source/CIGameScene.cpp

```diff
@@ -87,6 +87,21 @@ bool GameScene::init(const std::shared_ptr<cugl::AssetManager>& assets,
     // create the win scene
     _winScene = WinScene::alloc(assets, dimen);
 
+    // create the pause menu
+    _pauseMenu = PauseMenu::alloc(_assets, networkMessageManager, playerSettings);
+    _pauseMenu->setDisplay(false);
+
+    _pauseBtn = std::dynamic_pointer_cast<scene2::Button>(assets->get<scene2::SceneNode>("game_pausebutton"));
```

Comment:
Is there a reason to not have the pause button in the pause menu class?

Reply:
Isn't the pause button used to display the Pause Menu from the GameScene? like how settings button on the corner of menu screens is part of the Menu Scene.

Reply:
yeah but you still can make the pause button a field in the pause menu class